packages/web-components/src/checkbox/checkbox.styles.ts

```diff
@@ -0,0 +1,182 @@
+import { css } from '@microsoft/fast-element';
+import { display } from '@microsoft/fast-foundation';
+import {
+  borderRadiusCircular,
+  borderRadiusSmall,
+  colorCompoundBrandBackground,
+  colorCompoundBrandBackgroundHover,
+  colorCompoundBrandBackgroundPressed,
+  colorCompoundBrandForeground1,
+  colorCompoundBrandForeground1Hover,
+  colorCompoundBrandForeground1Pressed,
+  colorCompoundBrandStroke,
+  colorCompoundBrandStrokeHover,
+  colorCompoundBrandStrokePressed,
+  colorNeutralForeground1,
+  colorNeutralForeground2,
+  colorNeutralForeground3,
+  colorNeutralForegroundDisabled,
+  colorNeutralForegroundInverted,
+  colorNeutralStrokeAccessible,
+  colorNeutralStrokeAccessibleHover,
+  colorNeutralStrokeAccessiblePressed,
+  colorNeutralStrokeDisabled,
+  colorTransparentBackgroundHover,
+  fontFamilyBase,
+  fontSizeBase300,
+  lineHeightBase300,
+  spacingHorizontalS,
+  spacingHorizontalXS,
+  spacingVerticalS,
+  strokeWidthThin,
+} from '../theme/design-tokens.js';
+
+/** Checkbox styles
+ *
+ * @public
+ */
+export const styles = css`
+  ${display('inline-flex')}
+  :host {
+    align-items: center;
+    outline: none;
+    user-select: none;
+    vertical-align: middle;
+    pointer-events: pointer;
+  }
+  .control {
+    position: relative;
+    box-sizing: border-box;
+    display: flex;
+    flex-shrink: 0;
+    align-items: center;
+    justify-content: center;
+    overflow: hidden;
+    margin: ${spacingVerticalS} ${spacingHorizontalS};
+    fill: currentcolor;
+    height: 16px;
+    width: 16px;
+    border: ${strokeWidthThin} solid ${colorNeutralStrokeAccessible};
+    border-radius: ${borderRadiusSmall};
+    outline: none;
+  }
+  .label {
+    align-self: center;
+    cursor: inherit;
+    font-family: ${fontFamilyBase};
+    font-size: ${fontSizeBase300};
+    line-height: ${lineHeightBase300};
+    color: ${colorNeutralForeground3};
+    padding: ${spacingVerticalS} ${spacingHorizontalS} ${spacingVerticalS} ${spacingHorizontalXS};
+  }
+  .label__hidden {
+    display: none;
+    visibility: hidden;
+  }
+  .checked-indicator {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    color: ${colorNeutralForegroundInverted};
+    height: 16px;
+    font-size: 12px;
+    margin: auto;
+    opacity: 0;
+  }
+  .indeterminate-indicator {
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    width: 8px;
+    height: 8px;
+    transform: translate(-50%, -50%);
```

Comment:
Is this just to center it? Since it has a hight and a width and have you tried just using `inset: 0`? I think that would work, thn you can get rid of the `transform` `top` and `left`